_posts/2019-09-26-aws-waf-association-with-api-gateway.markdown

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title:  "AWS WAF association with API Gateway"
+date:   2019-09-26
+tags: AWS API WAF
+author: Natalie Laing
+image: img/lock.png
+---
+
+I was recently trying to attach a WAF (Web Application Firewall) regional ACL (Access Control List) to an  [API Gateway](https://whatis.techtarget.com/definition/API-gateway-application-programming-interface-gateway) using CloudFormation and I ran into problems when the API Gateway was created using Serverless Framework.
```

Comment:
I like JK's suggestion around the problems. Can you provide an example of the issue? IE.  a stack trace or similar...